core/ordering/cosipbft/mod.go

```diff
@@ -546,6 +547,7 @@ func (s *Service) doRound(ctx context.Context) error {
 
 	ctx, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
+	ctx = context.WithValue(ctx, tracing.ProtocolKey, blocksync.ProtocolName)
```

Comment:
Is it necessary ? `s.synx.Sync` will do the same.

Reply:
Nice catch! Removing this.